auth/src/main/java/io/grpc/auth/ClientAuthInterceptor.java

```diff
@@ -96,11 +104,29 @@ protected void checkedStart(Listener<RespT> responseListener, Metadata headers)
     };
   }
 
-  private Map<String, List<String>> getRequestMetadata() {
+  private URI serviceUri(Channel channel, MethodDescriptor<?, ?> method) throws StatusException {
+    String authority = channel.authority();
+    if (authority == null) {
+      throw Status.UNAUTHENTICATED.withDescription("Channel has no authority").asException();
+    }
+    final String suffixToStrip = ":443";
```

Comment:
When the variable has strong syntactic meaning to a variable, I see it counter-productive to make it a static variable. SUFFIX_TO_STRIP implies the current behavior in the method and vice versa; I see no value in splitting the two apart, and feel it is harmful.